### DISCUSSION TITLE
Bokepsin - Pop up google mengapa ini hanya terjadi di kiwi browser saja?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1511] [Q&A] [answered]
https://www.google.com/search?sca_esv=556639859&q=bokepsin+toge+bulat&tbm=isch&dpr=1.92#imgrc=b4IAlW1snAmfcM
![20230814_115613_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/26025563-4b89-48a8-8c1f-e7f6f480c351)

Pop up google mengapa ini hanya terjadi di kiwi browser saja

...

**Comment by richie3011** [comment, marked as answer]
> https://www.google.com/search?sca_esv=556639859&q=bokepsin+toge+bulat&tbm=isch&dpr=1.92#imgrc=b4IAlW1snAmfcM ![20230814_115613_rmscr.jpg](https://user-images.githubusercontent.com/114223791/260361409-26025563-4b89-48a8-8c1f-e7f6f480c351.jpg)
> 
> Pop up google mengapa ini hanya terjadi di kiwi browser saja
> 
> ...

Coba matiin adblock default kiwi

**Comment by Jokopentil** [comment]
👍 oke udah fi
